### ISSUE TITLE
Added Declarative Visualization to Krisk

### ISSUE THREAD

**Issue by napjon** [napjon/krisk#15]
Krisk users can just specify JSON option of Echarts to plot directly in Jupyter

**Comment by napjon** [comment]
User can just manually input without using krisk manipulation data. This already works. All that left is using adding to the notebooks.

```Python
from krisk.chart import Chart
chart = Chart()

chart.option['series'] = [{'data': [1, 1, 1, 1, 1], 'name': 'continent', 'type': 'bar'}]
chart.option['Xaxis'] =  {'data': ['Americas', 'Asia', 'Africa', 'Oceania', 'Europe']}

```

**Comment by napjon** [comment]
Fix in 0.2-develop, #45
